account_bank_statement_import_online/data/account_bank_statement_import_online.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" ?>
+<!--
+    Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+    Copyright 2019 Dataplug (https://dataplug.io)
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+
+    <record model="ir.cron" id="ir_cron_account_pull_online_bank_statements">
+        <field name="name">Pull Online Bank Statements</field>
+        <field name="interval_number">5</field>
```

Comment:
Better make this by default once every hour. At the most. The companies I work with usually import bank-statements once a day. Performance can suffer if there are too many very frequent cron jobs.

Reply:
Thanks, good point, adjusted.